PokeOneWeb/appsettings.json

```diff
@@ -1,7 +1,7 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Server=localhost\\SQLEXPRESS;Database=PokeOneWeb;Trusted_Connection=True;MultipleActiveResultSets=true;",
-    "ReadModelConnection": "Server=localhost\\SQLEXPRESS;Database=PokeOneWebReadModel;Trusted_Connection=True;MultipleActiveResultSets=true;"
+    "DefaultConnection": "Server=sql01,1433;Database=PokeOneWeb;User=sa;Password=1Secure*Password1;MultipleActiveResultSets=true;", //Trusted_Connection=True;
```

Comment:
Same as appsettings in PokeOneWeb.DataSync

Reply:
Would it be possible to add bot entries, so that if one is not available, the other gets used.